packages/unhead/src/scripts/proxy.ts

```
@@ -2,66 +2,39 @@ import type { AsVoidFunctions, RecordingEntry } from './types'
-      if (typeof v === 'object') {
-        return new Proxy(v, handler)
-      }
-      return v
-    },
-    apply(_, __, args) {
-      // does not return the apply output for consistency
-      // @ts-expect-error untyped
-      Reflect.apply(_, __, args)
-      return undefined
-    },
+    get(_, prop, receiver) { const v = Reflect.get(_, prop, receiver); return typeof v === 'object' ? new Proxy(v, handler) : v },
```

ESLint

[error] This line has 2 statements. Maximum allowed is 1.

(style/max-statements-per-line)